src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
```

Comment:
To improve the readability and understandability of your code, you can consider moving your shortened return statement in the arrow function to a new line and using round brackets, as shown below:
```
    ? people.filter((p) => (
        p.sex === 'm' && 
        Math.ceil(p.died / 100) === century
    ))
```
However, in your case, it might be excessive so it's up to you